### DISCUSSION TITLE
Prettify XML

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1138] [General]
I used "Prettify XML" and got some improvement, but this stays "different".
Why?
![grafik](https://user-images.githubusercontent.com/13277944/149767764-a5c99b57-3469-469d-bc18-d9d222e55ee1.png)

**Comment by sdottaka** [comment]
The Prettify XML plugin uses tidy-html5 to format XML files.

Unfortunately, I couldn't find options (http://api.html-tidy.org/tidy/quickref_5.4.0.html) to format `<Value></Value>` to `<Value />` or vice versa. 

Try using the DisplayXMLFiles.dll plugin instead.

https://github.com/WinMerge/winmerge/tree/master/Plugins/dlls/X64
